### ISSUE TITLE
use ccxt as exchange connector to open up a whole lot more exchanges

### ISSUE THREAD

**Issue by materemias** [benmarten/CryptoETF#54]
https://github.com/ccxt/ccxt
ccxt is an actively developed robust api wrapper, it would be worth to use to implement other exchanges fast

**Comment by benmarten** [comment]
Yes, CCXT could be a good choice for simplifying the setup part and integrating with a lot more exchanges... I am wondering though, if ccxt enforces a unique response format? (As format usually changes from each exchange, and that's kinda the main challenge that we are facing here)

**Comment by materemias** [comment]
ccxt has some functions which are implemented for all exchanges, and returns with the same structure. other unique api endpoints are reachable too but not standardised. try it, it is really easy to get started with.
